models/articles.models.js

```diff
@@ -10,7 +10,7 @@ exports.fetchArticles = () => {
   , articles.created_at
   , articles.votes
   , articles.article_img_url
-  , COUNT (*) AS comment_count
+  , COUNT (*)::INT AS comment_count
```

Comment:
Really nice addition here - do you know what the name for this is?

Reply:
I do not! I'll do some more research!